kube/stateful_set_test.go

```diff
@@ -156,25 +176,22 @@ func TestStatefulSetPorts(t *testing.T) {
 					skiff-role-name: myrole
 				type: ClusterIP
 		-
-			# This is the per-pod naming port
+			# This is the public service port
 			metadata:
-				name: myrole-pod
+				name: myrole-public
 			spec:
 				ports:
 				-
-					name: http
-					port: 80
-					# targetPort must be undefined for headless services
-					targetPort: 0
+						name: http
+						port: 80
+						targetPort: http
 				-
-					name: https
-					port: 443
-					# targetPort must be undefined for headless services
-					targetPort: 0
+						name: https
+						port: 443
+						targetPort: https
```

Comment:
Maybe we should update the test to have a private and a public port, to ensure that the differences are noted?